paths/graphql/01-introduction.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: simple-class
+header:
+  overlay_image: cover.jpeg
+  overlay_filter: rgba(46, 129, 200, 0.6)
+title: Introduction to GraphQL
+permalink: /graphql/
+next-page: /graphql/first-use
+facilitator: false
+sidebar:
+  nav: "github-graphql"
+main-content: |
+
+    When working on projects, it's common to want access to outside data. But data is often in flux, and that can be a big problem. Depending on your source, the data might change over the course of years, or merely within seconds.
+
+    Some examples of data you might want to work with could be:
+
+    - gathering the number of commits you made over a period of time
+    - flight statuses
+    - your favorite sports team's record, or
+    - weather in your area.
+
+    There are systems already in place that can enable you to gather up to date precise information that you want **and** immediately display it. Allow us to introduce you to APIs.
+
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/dsPVrbDHgaY?ecver=1" frameborder="0" allowfullscreen></iframe>
+
+    API stands for Application Program Interface, and many websites, software applications, and services use APIs to share and update information.
+
+    This course is going to introduce the concept of using APIs via the GitHub GraphQL Query Language, and to gather data and display it on a webpage. To find out more, click "Tell me why" below.
```

Comment:
I think we need to discuss REST APIs briefly here. The large majority of APIs are REST and GraphQL represents a significant change. I don't want to get too deep in the weeds with REST, but I think it needs a mention.